.gitignore

```diff
@@ -9,3 +9,4 @@ tests/.phpunit.result.cache
 vendor/
 .php_cs.cache
 \.idea/
+settings.json
```

Comment:
why?

Reply:
@ultrasites This is a VSCode settings file containing developer preferences which should not be committed.